### ISSUE TITLE
Update imgix support to be more granular (disable CP transforms by volume)

### ISSUE THREAD

**Issue by prescottcarr** [nystudio107/craft3-imageoptimize#384]
### Is your feature request related to a problem? Please describe.
CP requests are forcing the use of imgix and causing thumbnails from other Volumes (not using imgix) to 404/no display in the CP.

### Describe the solution you would like
It would be great if the integration of imgix could disable CP requests for certain transforms or be related to the Volume. For my example, I only use imgix for the 'Images' volume and it would be ideal if the CP would use imgix for these CP transforms (eg. User Photo Location) if the Volume is selected.

### Additional context

![Screenshot 2023-06-20 at 1 54 38 PM](https://github.com/nystudio107/craft-imageoptimize/assets/11970097/1a8dc76f-48d2-46f7-96fc-406bc1e2a6a8)

**Comment by robinbeatty** [comment]
+1 for support for choosing transformation pipeline per volume. In our case, Cloudinary is one filesystem and handles its own transforms via another plugin so when Sharp is enabled via OptimizeImages transforms in this volume fail. It's easy to conditionally fix this on the front-end but doesn't solve the CP thumbs from bouncing. 

I assume at some point your plugin is registering a transform route to Craft, which the other cloudinary plugin is doing too.. is there a cheap/short term fix you can think of that involves the changing order in which these are registered (I guess the last one registered takes precedence?)

**Comment by khalwat** [comment]
Yeah I'm not really sure what a great solution would be here, since the plugin isn't currently designed to handle different transform methods based on volumes.

Probably it would make more sense if there was a Cloudinary transform method built into ImageOptimize.
